retail install sections cleanup (#661)
SUSE/doc-susemanager: modules/retail/pages/retail-install-setup.adoc

Additionally, you will need to create a system group for each branch server, and each terminal hardware type in your environment.

You can create system groups using the {susemgr} {webui}.
Navigate to menu:Systems[System Groups] and click btn:[Create System Group].

For more information about system groups, see xref:reference:systems/system-groups.adoc[].

[[retail.sect.install.branch]]

> == Install the Branch Server and Register It to {susemgr}

Reviewer: ```suggestion
== Install the Branch Server and Register it to {susemgr}
```

For instructions to install Salt-based proxy clients and register them to {susemgr}, see xref:installation:install-proxy-unified.adoc[].

> The activation key should have the following channels, pools, and respective updates:

Reviewer: ```suggestion
The activation key should have these channels, pools, and respective updates:
```

When you are installing the branch server with a dedicated internal network, check that you are using the same fully qualified domain name (FQDN) on both the external and internal branch networks.
If the FQDN does not match on both networks, the branch server will not be recognized as a proxy server.

== Assign and Configure Branch Server Formulas

Before you configure the branch server, ensure you have decided on networking topology, and know the Salt ID of the branch server.

The branch server can be configured automatically using the [command]``retail_branch_init`` command, as shown in this section.
If you prefer to manually configure the branch server, you can do so using formulas.
For more information about formulas, see xref:retail:retail-formulas.adoc[].

> .Procedure: Configuring Branch Server Formulas Using Helper Script

Reviewer: ```suggestion
.Procedure: Configuring Branch Server Formulas With a Helper Script
```

> == Synchronize Images to the Branch Servers

Reviewer: ```suggestion
== Synchronize Images to the Branch Server
```

The OS image you use on the {susemgr} server must be synchronized for use on the branch server.
You can do this with the Salt [command]``image-sync`` tool.

> .Procedure: Synchronize Images with Branch Server

Reviewer: ```suggestion
.Procedure: Synchronizing Images to the Branch Server
```
